transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/internal/LogEvent.java

```diff
@@ -84,6 +90,12 @@ public abstract static class Builder {
     @NonNull
     public abstract Builder setNetworkConnectionInfo(@Nullable NetworkConnectionInfo value);
 
+    @NonNull
+    public abstract Builder setExperimentIds(@Nullable ExperimentIds value);
+
+    @NonNull
+    public abstract Builder setZwiebackCookieOverride(@Nullable String value);
```

Comment:
Why is this called Zwieback here but just CookieOverride in other SDKs? I'm also not familiar with the term.

Reply:
This is the name of the json field expected in the backend. To avoid confusion, I've simplified naming at API level.